### PR TITLE
User: show order detail

### PR DESCRIPTION
**Pull request by HangUet**
https://edu-redmine.framgia.vn/issues/415

### CODE REVIEW

**Comment by HangUet** [comment]
https://edu-redmine.framgia.vn/issues/415

**Comment by HangUet** [comment]
ready

File: app/views/user/orders/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<h1><%= t "order.submit.all_orders" %></h1>
+<% if @orders.blank? %>
```
Comment (huongnguyenmta) [inline app/views/user/orders/index.html.erb]: check điều kiện xong không xử lý gì à em?

**Comment by huongnguyenmta** [comment]
commented

**Comment by huongnguyenmta** [comment]
merged
merge ngen gái :-*

**Pull request merged**